.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.10.0-otp-22
```

Comment:
Do we want to bind ourselves to an `asdf` config? I would say the `mix.exs` specifies properly what Elixir version should be used to contribute. It's hard to assume what version manager people use to manage their elixir version. Just my two cents ✌️

Reply:
oh, actually no, I'll remove